### PR TITLE
Added Venator bow

### PR DESCRIPTION
**Pull request by Felris**
### Description:

Made Venator Bow creatable, chargeable, and added a small global boost, as well as boosts to Barb Assault, Whisperer, Grotesque Guardians, Bloodvelds, Mutated Bloodvelds, Suqahs, and Dagannoths. Boosts are based on feedback as well as osrs wiki. 

### Changes:

Added Venator Bow as a creatable for 5 Venator Shards. And Chargeable for 1 Ancient Essence per charge. 
Gives a 3% global boost when using Venator bow while training range.
Gives a 10% boost to Barbarian Assault.
Gives a 5% boost to Whisperer and Awakened Whisperer.
Gives a 5% boost to Grotesque Guardians.
Gives a 10% boost to Bloodvelds, and Mutated Bloodvelds.
Gives a 10% boost to Suqahs.
Gives a 10% boost to Dagannoths(not the kings).
TestPotato max gives 10,000 Venator Bow charges.

### Other checks:

- [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Comment by nwjgit** [comment]
be sure to run `yarn test:unit -u` to update the creatables unit test. also make sure the branch is up-to date with the main branch and that should fix the failing ge integration test.

File: src/lib/data/createables.ts [hunk 1]
```diff
@@ -2367,6 +2367,11 @@ const Createables: Createable[] = [
 			crafting: 35
 		}
 	},
+	{
+		name: 'Venator bow (uncharged)',
+		inputItems: new Bank().add('Venator shard', 5),
+		outputItems: new Bank().add('Venator bow (uncharged)')
```
Comment (gc) [inline src/lib/data/createables.ts:2373]: There's a mistake here.

Comment (Felris) [reply src/lib/data/createables.ts:2373]: What's the mistake here? Do you mean you want it to create the charged version? The way it works currently is it created the uncharged version and then when you /minion charge it becomes the normal version.

Comment (themrrobert) [reply src/lib/data/createables.ts:2373]: The only mistake I see here is that the Bank's don't end with .freeze(), however most of the creatables in that file don't have .freeze() at the end either.

eg. `inputItems: new Bank().add('Venator shard', 5).freeze(),`

File: src/lib/degradeableItems.ts [hunk 2]
```diff
@@ -218,6 +219,21 @@ export const degradeableItems: DegradeableItem[] = [
...
+	},
+	{
+		item: getOSItem('Venator bow'),
+		settingsKey: 'venator_bow_charges',
+		itemsToRefundOnBreak: new Bank().add('Venator bow (uncharged)').freeze(),
+		refundVariants: [],
+		setup: 'range',
+		aliases: ['venator bow, ven bow'],
+		chargeInput: {
+			cost: new Bank().add('ancient essence', 1).freeze(),
```
Comment (themrrobert) [inline src/lib/degradeableItems.ts:231]: Should be capital A in ancient essence

**Review by themrrobert — CHANGES_REQUESTED** [review]

File: src/lib/degradeableItems.ts [hunk 3]
```diff
@@ -218,6 +219,21 @@ export const degradeableItems: DegradeableItem[] = [
...
 		convertOnCharge: true,
 		emoji: ''
+	},
+	{
+		item: getOSItem('Venator bow'),
+		settingsKey: 'venator_bow_charges',
+		itemsToRefundOnBreak: new Bank().add('Venator bow (uncharged)').freeze(),
+		refundVariants: [],
+		setup: 'range',
+		aliases: ['venator bow, ven bow'],
```
Comment (gc) [inline src/lib/degradeableItems.ts:229]: this is accidentally put as 1 string instead of 2 strings

**Pull request merged**